src/parse-stream.js

```diff
@@ -103,6 +104,11 @@ export default class ParseStream extends Stream {
       return;
     }
 
+    // map tag
```

Comment:
is there any danger in having both a mapper and a custom parser on the same tag?

i.e you already have a custom parser on `#VOD-TIMING` and you want that to continue to work

Reply:
If you have a mapper that converts `#VOD-TIMING` to something else and you have a parser that extracts data from `#VOD-TIMING` then the parser will not work. Do we want the old tag to still exist?

Reply:
I think we'd want the custom parsers to still work. Since custom parsers put stuff in `segment.custom` it would probably be ok for the custom parsers to run before the tag mappers, right? That way the old behavior continues to work:

```
const manifest = '#VOD-TIMING:wallclock=1511816599485';

var parser = new m3u8Parser.Parser();

parser.addParser({
  expression: /^#VOD-TIMING/,
  customType: 'timing'
});

parser.addTagMapper({
  expression: /#VOD-TIMING/,
  map(line) {
    const regex = /#VOD-TIMING:(?:wallclock=([^,\n]+))/g;
    const match = regex.exec(line);
    if (match) {
      const ISOdate = new Date(Number(match[1])).toISOString();
      return `#EXT-X-PROGRAM-DATE-TIME:${ISOdate}`;
    }
    return line;
  }
});

parser.push(manifest);
parser.end();

parser.manifest.custom.timing; // 1
parser.manifest.segments[0].dateTimeObject; // #VOD-TIMING:1511816599485
```

Reply:
Is there a use case for this aside from easier debugging?

Reply:
uh, well it's possible that a user that writes some plugin/source handler adds a tagParser that then breaks a different downstream user.

It's mostly for backwards compatibility.